bin/classify_SingleR.R

```diff
@@ -21,6 +21,12 @@ option_list <- list(
     help = "path to file containing a single model generated for SingleR annotation. 
             File name is expected to be in form: <model name>_model.rds."
   ),
+  make_option(
+    opt_str = c("--singler_label_name"),
+    type = "character",
+    default = "label.ont",
+    help = "label used when building the SingleR reference"
+  ),
```

Comment:
For the purposes of this script, I think it makes more sense for this to be a flag rather than a specific label, as the label name is not part of the output object. So this is a very specific case for something that might be a more general use, and doesn't quite signal why we have this option.

```suggestion
  make_option(
    opt_str = c("--singler_uses_ontology"),
    action = "store_true",
    default = FALSE,
    help = "Indicates that the SingleR reference labels in the model are ontology ids. SingleR model object should include `cell_ontology_df`."
  ),
```

You can then test that the second sentence there is true, and error if not.

Reply:
> You can then test that the second sentence there is true, and error if not.

Actually... the presence of `cell_ontology_df` is something we can check for and just eliminate the option altogether!